src/ui/settings/index.js

```diff
@@ -29,7 +29,8 @@ export default function setupSettings(kbnServer, server, config) {
     return client
       .get({ ...clientSettings })
       .then(res => res._source)
-      .then(user => hydrateUserSettings(user));
+      .then(user => hydrateUserSettings(user))
+      .catch(e => { return {}; });
```

Comment:
Returning an object from an arrow function is a fairly common scenario, so I'd expect most people to be aware of what's going on (or be able to infer it from context at least).

By the way I would move the `.catch` call one level up, so that it doesn't muck with `hydrateUserSettings`, which has no expected reason for throwing even if an ES error is caught.